packages/permission-controller/src/rpc-methods/getPermissions.test.ts

```diff
@@ -9,6 +10,7 @@ describe('getPermissions RPC method', () => {
     });
 
     const engine = new JsonRpcEngine();
+    // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
Why did it have trouble with this?

Reply:
> ESLint: Promise returned in function argument where a void return was expected.(@typescript-eslint/no-misused-promises)

It's because the implementation _can_ return a Promise, which would be unhandled otherwise. There may be a better solution here.